leptos_workers_macro/src/v2/analyze.rs

```diff
@@ -34,12 +34,15 @@ const VALID_SIGNATURES: &str = indoc! { r"
         [pub] [async] fn worker(req: Request, callback: impl Fn(Response))
     Channel:
         [pub] [async] fn worker(rx: leptos_workers::Receiver<Request>, tx: leptos_workers::Sender<Response>)
+        or with an initialisation parameter:
+        [pub] [async] fn worker(init: Init, rx: leptos_workers::Receiver<Request>, tx: leptos_workers::Sender<Response>)
```

Comment:
```suggestion
        [pub] [async] fn worker([init: Init,] rx: leptos_workers::Receiver<Request>, tx: leptos_workers::Sender<Response>)
```
since that's how optional pub/async is represented anyways